config/68-azure-sriov-nm-unmanaged.rules

```diff
@@ -0,0 +1,4 @@
+# Accelerated Networking on Azure exposes a new SRIOV interface to the VM.
+# This interface is transparently bonded to the synthetic interface,
+# so NetworkManager should just ignore any SRIOV interfaces.
+SUBSYSTEM=="net", DRIVERS=="hv_pci", ACTION=="add", ENV{NM_UNMANAGED}="1"
```

Comment:
```suggestion
SUBSYSTEM=="net", DRIVERS=="hv_pci", ACTION=="add|change|move", ENV{NM_UNMANAGED}="1"
```

I was testing out a version of this over in https://github.com/coreos/fedora-coreos-config/pull/2176 and wasn't observing `NM_UNMANAGED=1` getting set in the initramfs until I added the `change|move`, which is what is done over in https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/blob/38d3834e2c464bd4820392800a3a1fe3f286fc62/data/85-nm-unmanaged.rules#L5

Reply:
@lkundrak WDYT?

Reply:
I would NOT recommend this change without a driver check (mlx4, mlx5).  There is potential for hv_pci devices to be managed.  We're working on a better story wrt MANA: https://learn.microsoft.com/en-us/azure/virtual-network/accelerated-networking-mana-linux

Reply:
@cjp256 do you have an example modification to this rule to achieve that goal?